CHANGELOG

```diff
@@ -9,3 +9,15 @@
   * vulnerability: replace limit states lowercased names with themselves where
     all [^a-z0-9] chars are substituted with '_' and contiguous '_' are
     collapsed into one
+
+  [Ben Wyss]
+  * Revers the x / y axis of the fragility discrete tables.
+
+  * Add intensity measure as a column for the fragility discrete tables.
+
+  * Add intensity measure as a column for the vulnerability discrete tables.
+
+  * Vulnerability discrete table header is generated dynamically when the user selects an IML from a provided selection menu.
+
+  * If the selected IML for a discrete vulnerability function is "SA" then the user is prompted to provide a value for the SA period before the table is generated.
+
```

Comment:
Sintax is not fully compliant our format:
  * no white lines between items of the same developer
  * lines length no more than 80 characters
Here a full description: https://github.com/gem/oq-engine/blob/master/doc/engine/Developers-Notes.md#guidelines-for-debianchangelog-file